web-console/1-homepage/search.md

```diff
@@ -1,50 +1,57 @@
-# Template
+# Search
 
-- This is a description of the section and where it is located/how to get to it
-- This is a description of what this section is for
+- The search bar is located at the very top of the web console homepage
+- This component enables users to enter a query and receive a list of results.
 
 
-## Use Case 1 (e.g. Browse for service)
+## Search for Service
```

Comment:
This is actually Search the Catalog ... it will contain both builder images and services @cshinn